gxa/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentPageService.java

```diff
@@ -88,7 +93,7 @@ private JsonElement experimentDescription(Experiment experiment,
         return experimentDescription;
     }
 
-    private URI callbackLink(String urlBase, String accessKey, Pair<String, String>... params) {
+    private URI callbackLink(String urlBase, String accessKey, List<Pair<String, String>> params) {
```

Comment:
Why does this method need to take a list? It is only ever called with a single Pair or no pair. So an `Optional<Pair>`?